eng/native/configureplatform.cmake

```diff
@@ -118,6 +118,11 @@ if(CLR_CMAKE_HOST_OS STREQUAL FreeBSD)
     set(CLR_CMAKE_HOST_UNIX 1)
     set(CLR_CMAKE_HOST_UNIX_AMD64 1)
     set(CLR_CMAKE_HOST_FREEBSD 1)
+
+    if (${CMAKE_HOST_SYSTEM_NAME} STREQUAL Linux)
+        set(CLR_CMAKE_HOST_OS Linux)
```

Comment:
This is wrong.  `FreeBSD` is not a `Linux` variant. `Host` represents the machine where the generated code will run.  For your scenario, I don't understand why `Host` and `Target` are not the same.